using-mobile-extensions/adobe-target-vec.md

```diff
@@ -0,0 +1,144 @@
+
+ @Override
+ public void onCreate() {
+     super.onCreate();
+     MobileCore.setApplication(this);
+     try {
+         //Other Extensions that you need
+         TargetVEC.registerExtension();
+         Target.registerExtension();
+     } catch (Exception e) {
+         //Log the exception
+     }
+ }
+}
+```
+{% endtab %}
+
+{% tab title="iOS" %}
+#### Objective-C
+
+1. In your app's `didFinishLaunchingWithOptions` function register the Target VEC extension
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  //Other Extensions that you need
+  [ACPTargetVEC registerExtension];
+  [ACPTarget registerExtension];
+
+  // Override point for customization after application launch.
+  return YES;
+}
+```
+
+#### Swift
+1. In your app's `didFinishLaunchingWithOptions` function register the Target VEC extension
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  //Other Extensions that you need
+  ACPTargetVEC.registerExtension()
+  ACPTarget.registerExtension()
+  
+  return true
+}
+```
+{% endtab %}
+{% endtabs %}
+
+## Implementation Methods for Target VEC   <a id="implementation-methods-for-target-vec"></a>
+The Target VEC extension retrieves the relevant Target experiences for your app through a single network request. Offers are retrieved via this network call and applied automatically on the targeted screens. It should be noted that no network requests are made to retrieve VEC experiences as the user navigates through multiple screens of the app.
+
+The default behavior of the extension is to make a synchronous network request (blocking call) at the time of Application launch. You can use Launch to control the behavior of this network request to meet your application behavior.
+
+### Auto-Fetch Target Activities
+This is the default behavior where a network request is initiated automatically by the Target VEC extension. You can use one of the following options to make this request a blocking call or an asynchronous request.
+
+#### Fetch in a synchronous call (background is OFF)
+When selected Target VEC extension makes a network request as a blocking call on App launch. Offers are applied immediately and there is no flicker in the app. This is the default behavior of the extension.
+
+#### Fetch in an asynchronous call (background is ON)
+When selected Target VEC extension makes a network request in the background on App launch, but does not block the app from loading. If your experiences are authored on the home screen of your app, the offers may not apply for the home screen if the screen is rendered (this is typically when the lifecycle events `didFinishLaunchingWithOptions` and `onActivityResumed`  are triggered in iOS and Android respectively) before the call completes. Offers will be automatically apply on all subsequent screens.
```

Comment:
please add a comma between "selected" and "Target VEC extension"

Reply:
It changes the meaning. I have edit it to this:
When selected, Target VEC extension makes a network request in the background on App launch but does not block the app from loading. If your experiences are authored on the home screen of your app, the offers may not apply for the home screen if the screen is rendered before the call completes. The app screen rendering is typically identified via the lifecycle events `didFinishLaunchingWithOptions` and `onActivityResumed`  on iOS and Android respectively. Offers will be automatically apply on all subsequent screens.